### PR TITLE
hello-thirdparty: fix lint warning and rename package

### PR DESCRIPTION
**Pull request by proppy**
Fixes the following lint warnings:
```
Correctness
4	Warning GradleDynamicVersion: Gradle Dynamic Version
Usability:Icons
1	Warning MissingApplicationIcon: Missing application icon
```
Related #82

### CODE REVIEW

File: hello-thirdparty/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- BEGIN_INCLUDE(manifest) -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-        package="com.example.android.native_playgameservices"
-        android:versionCode="1"
-        android:versionName="1.0">
+          package="com.example.android.hello_thirdparty"
+          android:versionCode="1"
+          android:versionName="1.0">
+  <!-- This .apk has no Java code itself, so set hasCode to false. -->
```
Comment (claywilkinson) [inline hello-thirdparty/app/src/main/AndroidManifest.xml]: does it need hasCode = false, or should the comment be removed?

Comment (proppy) [inline hello-thirdparty/app/src/main/AndroidManifest.xml]: The comment should be removed.

Comment (proppy) [inline hello-thirdparty/app/src/main/AndroidManifest.xml]: added hasCode

**Comment by proppy** [comment]
PTAL

**Comment by claywilkinson** [comment]
LGTM

**Pull request merged**